en-US/emails/2017/copyright.lang

```diff
@@ -0,0 +1,81 @@
+## NOTE: Email to remind people we’re doing the campaign and ask them to share their story
+
+
+# Subject candidate 1
+;Share your story —
+Share your story —
```

Comment:
So, it's either "Subject 1 + preheader" or "Subject2"? I would add it to the note anyway